ops/docker/Dockerfile.replica-healthcheck

```diff
@@ -0,0 +1,32 @@
+FROM ethereumoptimism/builder AS builder
+
+FROM node:14-alpine
+
+RUN apk add --no-cache curl bash jq
+
+WORKDIR /opt/optimism
+
+# copy top level files
+COPY --from=builder /optimism/*.json ./
+COPY --from=builder /optimism/yarn.lock .
+COPY --from=builder /optimism/node_modules ./node_modules
+
+# copy deps (would have been nice if docker followed the symlinks required)
+COPY --from=builder /optimism/packages/core-utils/package.json ./packages/core-utils/package.json
+COPY --from=builder /optimism/packages/core-utils/dist ./packages/core-utils/dist
+COPY --from=builder /optimism/packages/common-ts/package.json ./packages/common-ts/package.json
+COPY --from=builder /optimism/packages/common-ts/dist ./packages/common-ts/dist
+
+COPY --from=builder /optimism/packages/contracts/package.json ./packages/contracts/package.json
+COPY --from=builder /optimism/packages/contracts/deployments ./packages/contracts/deployments
+COPY --from=builder /optimism/packages/contracts/dist ./packages/contracts/dist
+COPY --from=builder /optimism/packages/contracts/artifacts ./packages/contracts/artifacts
+COPY --from=builder /optimism/packages/contracts/artifacts-ovm ./packages/contracts/artifacts-ovm
+
+# copy the service
+WORKDIR /opt/optimism/packages/replica-healthcheck
+COPY --from=builder /optimism/packages/replica-healthcheck/dist ./dist
+COPY --from=builder /optimism/packages/replica-healthcheck/package.json .
+COPY --from=builder /optimism/packages/replica-healthcheck/node_modules ./node_modules
+
+ENTRYPOINT ["node", "dist/src/exec/run-healthcheck-server.js"]
```

Comment:
`ENTRYPOINT ["node", "dist/src/exec/run-healthcheck-server.js"]`
should be
`ENTRYPOINT ["node", "dist/exec/run-healthcheck-server.js"]` ?

Reply:
good catch! fixed and tested locally that this runs from the correct entrypoint